components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/install_utils.cc

```diff
@@ -3,40 +3,34 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at https://mozilla.org/MPL/2.0/. */
 
-#include "brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/service_utils.h"
+#include "brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/install_utils.h"
 
 #include "base/base_paths.h"
 #include "base/command_line.h"
 #include "base/files/file_path.h"
+#include "base/files/file_util.h"
 #include "base/logging.h"
 #include "base/path_service.h"
+#include "base/win/windows_types.h"
 #include "brave/components/brave_vpn/browser/connection/common/win/scoped_sc_handle.h"
 #include "brave/components/brave_vpn/browser/connection/common/win/utils.h"
-#include "brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/service_constants.h"
+#include "brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/common/service_constants.h"
+#include "brave/components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/common/wireguard_utils.h"
 #include "chrome/installer/util/install_service_work_item.h"
 
 namespace brave_vpn {
 
-bool InstallBraveWireguardService() {
-  base::FilePath exe_dir;
-  base::PathService::Get(base::DIR_EXE, &exe_dir);
-  base::CommandLine service_cmd(
-      exe_dir.Append(brave_vpn::kBraveVpnWireguardServiceExecutable));
-  installer::InstallServiceWorkItem install_service_work_item(
-      brave_vpn::GetBraveVpnWireguardServiceName(),
-      brave_vpn::GetBraveVpnWireguardServiceDisplayName(), SERVICE_AUTO_START,
-      service_cmd, base::CommandLine(base::CommandLine::NO_PROGRAM),
-      brave_vpn::GetBraveVpnWireguardServiceRegistryStoragePath(),
-      {brave_vpn::GetBraveVpnWireguardServiceClsid()},
-      {brave_vpn::GetBraveVpnWireguardServiceIid()});
-  install_service_work_item.set_best_effort(true);
-  install_service_work_item.set_rollback_enabled(false);
-  if (install_service_work_item.Do()) {
-    return brave_vpn::ConfigureBraveWireguardService(
-        brave_vpn::GetBraveVpnWireguardServiceName());
+namespace {
+bool RemoveWireguardConfigDirectory(const base::FilePath& last_used_config) {
+  auto wireguard_config_folder = last_used_config.DirName();
+  if (wireguard_config_folder.value().empty() ||
```

Comment:
nit:  can check directly with `empty()`
```suggestion
  if (wireguard_config_folder.empty() ||
```